packages/components/tabs/src/tab-group.ts

```diff
@@ -351,6 +351,10 @@ export class TabGroup extends ScopedElementsMixin(LitElement) {
     if (selectedTab !== this.selectedTab) {
       this.tabs?.forEach(tab => tab.toggleAttribute('selected', tab === selectedTab));
 
+      if (selectedTab.renderRoot.querySelector('a[href]')) {
+        (selectedTab.renderRoot.querySelector('a[href]') as HTMLAnchorElement).click();
```

Comment:
Should this not be done on the `<sl-tab>` component itself?

Reply:
So the tab itself probably doesn't know a user used the keyboard to select it. You can check if a link is present using `selectebTab.href`. 

But perhaps you could just add a `keydown` event listener to `<sl-tab>`? And listen for space/enter etc.?

Reply:
We are adding possibility to use href there, so on click href just changes the url by default, but it's not working when we're using keyboard navigation there, in my opinion it should be possible.

Reply:
I agree, but I think that should be done in tab, not in the tab-group component :)